web-api/src/applicationContext.ts

```diff
@@ -97,20 +96,8 @@ const entitiesByName = {
   WorkItem,
 };
 
-export const createApplicationContext = (
-  appContextUser = {},
-  logger = createLogger(),
-) => {
-  const user = new User(appContextUser);
-
-  if (process.env.NODE_ENV === 'production') {
-    const authenticated = user && Object.keys(user).length;
-    logger.defaultMeta = logger.defaultMeta || {};
-    logger.defaultMeta.user = authenticated
-      ? user
-      : { role: 'unauthenticated' };
-  }
-
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const createApplicationContext = (appContextUser = {}) => {
```

Comment:
Is there a good reason not to get rid of `appContextUser` besides a slightly larger git diff?

Reply:
Good Question. No that is the only reason.

If I removed it I would need to update 50 files and I wanted this PR to really show what was changing for the logger.